druntime/src/core/exception.d

```diff
@@ -608,7 +608,7 @@ extern (C) void onUnittestErrorMsg( string file, size_t line, string msg ) nothr
  * Throws:
  *  $(LREF RangeError).
  */
-extern (C) void onRangeError( string file = __FILE__, size_t line = __LINE__ ) @trusted pure nothrow @nogc
+extern (C) void onRangeError( string file = __FILE__, size_t line = __LINE__ ) @trusted pure nothrow @nogc noreturn
```

Comment:
`noreturn` is a type, not a function attribute

Reply:
Ahh, will adjust.

Reply:
Done.

Reply:
Ahh, this opens a can of worms of error messages. I didn't think of that.

Reply:
Green now.